app/columns.tsx

```diff
@@ -139,8 +141,10 @@ export const fxmsMessageColumns: ColumnDef<FxMSMessageInfo>[] = [
         return ( <></> );
       }
 
-      if (props.row.original.ctrDashboardLink) {
-        return OffsiteLink(props.row.original.ctrDashboardLink, "Dashboard");
+      if (props.row.original.ctrDashboardLink && props.row.original.ctrPercent !== undefined && props.row.original.template !== 'infobar') {
+        return OffsiteLink(props.row.original.ctrDashboardLink, props.row.original.ctrPercent + "% CTR");
+      } else if (props.row.original.ctrDashboardLink) {
+        return OffsiteLink(props.row.original.ctrDashboardLink, "Dashboard")
```

Comment:
The above clause looks identical to the next one except for the exclusion of infobar CTRs.  If they're broken at the moment, don't we want to exclude them both places?

If so, I'd suggest hoisting this code into its own function. Please also include a link to the bug about enabling inline CTR percentages for infobar.